pkg/dns/proto/nds.pb.go

```diff
@@ -1,68 +1,94 @@
+// Copyright Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Generated code probably shouldn't have this header.

Reply:
PRs welcome if you want to deal with the legal headaches..

Reply:
What legal headaches? This is explicitly called out that standard protoc output does not need a legal header.